src/reference/asciidoc/index.adoc

```diff
@@ -5,6 +5,14 @@
 :numbered:
 :icons: font
 :hide-uri-scheme:
+:spring-amqp-version: 2.1.3
```

Comment:
That's not good. No one is going to support this variable here.
We have a Gradle on top of project build, so would be great to propagate its `project-version` over here into the docs as well.

In Spring Integration we did it like this:
```
[source, groovy, subs="normal"]
----
compile "org.springframework.integration:spring-integration-amqp:{project-version}"
----
```

Hope it can help somehow...
`subs="normal"` is a trick over here.

Reply:
You don't need to sent anything from the Gradle file.
According their docs the `project-version` attribute matches a `$project.version` variable: https://asciidoctor.org/docs/asciidoctor-gradle-plugin/

Reply:
That took a little doing, but I got it. It turns out that I had to pass the build.gradle file itself to the Asciidoctor attributes list. That seems odd to me, but it works (and is what the Asciidoctor folks say to do).